app/web/src/components/FuncEditor/AttributeBindings.vue

```diff
@@ -200,9 +200,8 @@ const prototypeViews = computed(() =>
 
       const args = proto.prototypeArguments.map((arg) => ({
         name: funcArgumentsIdMap?.value[arg.funcArgumentId]?.name ?? "none",
-        prop: arg.inputSocketId
-          ? funcStore.inputSocketIdToSourceName(arg.inputSocketId) ?? "none"
-          : "none",
+        // TODO(nick): re-evaluate this when prop id is added back to the attribute prototype argument bag.
+        prop: arg.inputSocketId ?? "none",
```

Comment:
You'll see a few changes like this. Essentially, the `AttributePrototypeArgumentBag` that is sent down needs a way to tell users "hey, I want my input from a `Prop`". In the old engine, it didn't need to do this. In fact, it had to do something worse: it found the implicit `InternalProviders` (remember those?) and then mapped them back to the `Props`. Now, we can just use the `Props` directly... just not in this PR due to its size and scope.